src/VecSim/algorithms/brute_force/brute_force.h

```diff
@@ -33,7 +33,7 @@ class BruteForceIndex : public VecSimIndexAbstract<DistType> {
                                                     VecSimQueryResult_Code *rc) const;
     virtual VecSimQueryResult_List topKQuery(const void *queryBlob, size_t k,
                                              VecSimQueryParams *queryParams) override;
-    VecSimQueryResult_List rangeQuery(const void *queryBlob, DistType radius,
+    VecSimQueryResult_List rangeQuery(const void *queryBlob, double radius,
```

Comment:
Please remind me why we moved to `double` from `DistType`?

Reply:
VecSimIndexInterface is not templates, so its inheritors must implement exactly VecSimQueryResult_List rangeQuery(... double radius)
hence for floats it gives an error